pkg/config/handler.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// Handler returns the current configuration as json when hitting the /config url
+func Handler(c *Config) http.HandlerFunc {
```

Comment:
Let's keep `pkg/config` non-http-aware, and move this file to `cmd/proxy/actions`. We can call it `actions.ConfigHandler`. 

This helps keep all of our handler in one place (of course with exception to the download.Protocol handlers since they are tightly coupled with the logic of a GOPROXY)